docs/docs/validation-action.mdx

```diff
@@ -0,0 +1,246 @@
+    send messages back to the user. Use
+    `dispatcher.utter_message()` or any other
+    `rasa_sdk.executor.CollectingDispatcher`
+    method. See the [documentation for the dispatcher](sdk-dispatcher.mdx)
+
+  * **tracker** – the state tracker for the current
+    user. You can access slot values using
+    `tracker.get_slot(slot_name)`, the most recent user message
+    is `tracker.latest_message.text` and any other
+    `rasa_sdk.Tracker` property. See the [documentation for the tracker](sdk-tracker.mdx).
+
+  * **domain** – the bot's domain
+
+###### **Returns**
+
+  A list of `rasa_sdk.events.Event` instances. See the [documentation for events](sdk-events.mdx).
+
+###### **Return type**
+
+  `List`[`Dict`[`str`, `Any`]]
+
+##### ValidationAction.required_slots
+
+```python
+async ValidationAction.required_slots(domain_slots, dispatcher, tracker, domain)
+```
+
+The `required_slots` method will return the `domain_slots` which is a list of all slot names mapped in the domain that
+do not include any slot mapping with conditions. `domain_slots` is returned by the `domain_slots` method, which only takes
+`Domain` as an argument.
+
+###### **Returns**
+
+  A list of slot names of type `Text`.
+
+##### ValidationAction.get_extraction_events
+
+```python
+async ValidationAction.get_extraction_events(dispatcher, tracker, domain)
+```
+
+The `get_extraction_events` method will gather the list of slot names via `required_slots` method call and then loop
+through every slot name to run the `extract_<slot name>` method if available.
+
+##### **Returns**
+
+  A list of `rasa_sdk.events.SlotSet` instances. See the [documentation for SlotSet events](sdk-events.mdx#slotset).
+
+##### ValidationAction.get_validation_events
+
+```python
+async ValidationAction.get_validation_events(dispatcher, tracker, domain)
+```
+
+The `get_validation_events` method will gather the list of slot names to validate via `required_slots` method call.
+Then it will get a mapping of slots which were recently set and their values via `tracker.slots_to_validate` call.
+Looping through this mapping of recently extracted slots, it will check if the slot is in `required_slots` then run the
+`validate_<slot name>` method if available for that slot.
+
+###### **Returns**
+
+  A list of `rasa_sdk.events.SlotSet` instances. See the [documentation for SlotSet events](sdk-events.mdx#slotset).
+
+## `FormValidationAction` class
+
+A `FormValidationAction` custom action will only run if the form it specifies in its name is activated.
+If certain custom slot mappings should only be extracted and / or validated within the context of a form,
+the custom action should inherit from `FormValidationAction` rather than from `ValidationAction`.
+Since `FormValidationAction` runs on every user turn only as long as the form is active, it is not required to use
+mapping conditions for custom slot mappings defined in a custom action extending `FormValidationAction`.
```

Comment:
```suggestion
Since  a custom action extending `FormValidationAction` runs on every user turn only as long as the form is active, it is not required to use
mapping conditions in this case.
```